docs/the-contribution-workflow.md

```diff
@@ -0,0 +1,245 @@
+## Contributing to Exercism
+
+Open source contributions can be intimidating! As you're looking around, wondering where to dive in, you might think everyone _else_ knows what's going on in the code base, or has better grasp of the language, or seems to be well known around the project.
+
+As a total beginner, where do you begin?
+
+Right here. This guide will walk you through your first open source contribution.
+
+It's going to target the smallest possible fix (improving documentation), as a "bridge" to more complex contributions later.
+
+First, go take a look at [Contributing to Exercism.io](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md#submitting-code-changes)
+
+If not all of that makes sense to you, keep reading. This is for you.
+
+
+## Table of Contents
+
+- [Find the right contribution](#find-the-right-contribution)
+- [Claiming that contribution](#claiming-that-contribution)
+- [Working on the code](#working-on-the-code)
+- [Submitting your contribution](#submitting-your-contribution)
```

Comment:
Throughout the guide, it might make sense to point to other documents in the docs repo, as "deep dives" on a subject.